### ISSUE TITLE
Player.attachTo sync is broken

### ISSUE THREAD

**Issue by litolax** [altmp/altv-issues#2178]
### Description of the problem

When I attach player to player sync for player who we attach is broken

https://github.com/altmp/altv-issues/assets/60060712/d5c92deb-9598-4e79-be68-927b4d73704e

### Reproduction steps

```js
alt.Player.all[1].attachTo(player, 127, 127, new alt.Vector3(0.27, 0.15, 0.63), new alt.Vector3(0,0,0),
false, false);
```

### Expected behaviour

Sync should work correctly for all players

### Operating system

Windows 11

### Version

16.0.0-dev.114 (dev)

### Confirmation of issue's presence

- [X] *By submitting this ticket, I affirm that I have verified the presence of this issue on the latest developer version available at the time of writing this ticket.*

**Comment by litolax** [comment]
https://github.com/altmp/altv-issues/assets/60060712/86423bcc-e04b-46e9-8e2e-98942d39d0c9

Works fine on v15 release

**Comment by litolax** [comment]
Fixed with the latest 16.0.0-dev.132 (dev)
